mico-core/src/main/java/io/github/ust/mico/core/dto/GitHubCrawlingInfoDTO.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.github.ust.mico.core.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import io.github.ust.mico.core.service.GitHubCrawler;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.Accessors;
+
+
+/**
+ * DTO for the information needed by the {@link GitHubCrawler}
+ * for crawling a service from a GitHub repository.
+ */
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Accessors(chain = true)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class GitHubCrawlingInfoDTO {
+
+    /**
+     * Link to the GitHub repo to crawl from (required).
+     */
+    private String uri;
+
+    /**
+     * The GitHub release tag. Defaults to 'latest'.
+     */
+    private String version = "latest";
```

Comment:
Consider to use annotation `@NotEmpty` to ensure that it won't be set to null or to an empty string.

Reply:
Already added, commit coming in a few seconds